portal-web/docroot/errors/code.jsp

```diff
@@ -43,6 +43,9 @@ String xRequestWith = request.getHeader(HttpHeaders.X_REQUESTED_WITH);
 %>
 
 <c:choose>
+	<c:when test="<%= GetterUtil.getBoolean(request.getAttribute(WebKeys.UNKNOWN_VIRTUAL_HOST)) %>">
+		Unknown virtual host name : <%= PortalUtil.getHost(request) %>
```

Comment:
@david-truong if you want some one from front end to make the 404 page prettier, do it here.